iotedgedev/envvars.py

```diff
@@ -109,7 +109,7 @@ def load(self, force=False):
                 self.RUNTIME_CONFIG_DIR = self.get_envvar("RUNTIME_CONFIG_DIR", default=".")
                 if self.RUNTIME_CONFIG_DIR == ".":
                     self.set_envvar("RUNTIME_CONFIG_DIR", self.get_runtime_config_dir())
-                self.ACTIVE_MODULES = self.get_envvar("ACTIVE_MODULES")
```

Comment:
This is a breaking change, so we have to make sure this is documented in the release notes.  Please make a note somewhere or file an issue to track these for the next release.  THanks,

Reply:
Sure. Issue filed: https://github.com/Azure/iotedgedev/issues/201